ow_power_system/src/power_system_node.cpp

```diff
@@ -29,22 +29,42 @@ int main(int argc, char* argv[]) {
   ros::Publisher RUL_pub = n.advertise<std_msgs::Float64>("remaining_useful_life",1000);
   
   //Load power values csv
-  string csv_path,csv_file;
-  csv_file = "/data/onewatt.csv";
-  //FILE OPTIONS: onewatt.csv, eightwatt.csv, sixteenwatt.csv
-  csv_path = ros::package::getPath("ow_power_system");
-  csv_path += csv_file;
+  string csv_path;
+  auto csv_path_param_exist = n.param("power_draw_csv_path", csv_path,
```

Comment:
A more descriptive variable name than 'n' would be nice, even as short as 'nh' for node handle.

Reply:
I will address shortly

Reply:
addressed